packages/cli/src/utils/verifyManifest.ts

```diff
@@ -26,24 +26,25 @@ const secretSchema = object({
 });
 const secretsArraySchema = array(secretSchema);
 
-const proxyOptionsSchema = object({
+const requestOptionsSchema = object({
     method: z.enum(['GET', 'POST', 'PUT', 'DELETE']),
-    headers: z.record(string()),
-    body: z.any(),
+    headers: z.record(string()).optional(),
+    body: z.any().optional(),
```

Comment:
Maybe add a test case to cover these two new features? I don't think there's one for not throwing an error on a request that has no headers or body

Reply:
Added additional test case